src/App.js

```diff
@@ -1,16 +1,51 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog'; 
+
 
 const App = () => {
+
+  const [chatMessageList, setChatMessageList] = useState(chatMessages);
+
+  const likedChatMessage = (id) => {
+    setChatMessageList(chatMessageList => chatMessageList.map(message => {
+      if(message.id === id) {
+        return {...message, liked: !message.liked};
+      } else {
+        return message;
+      }
+    }));
+  };
+
+
+  const likedCounter = (chatMessageList) => {
+    return chatMessageList.reduce((total, message) => {
+      if (message.liked) {
+        return total + 1;
+      } else {
+        return total;
+      }
+    }, 0);
+  };
+
+  const totalLiked = likedCounter(chatMessageList)
+
   return (
-    <div id="App">
+    <div id='App'>
       <header>
-        <h1>Application title</h1>
+        <h1>Chatter box</h1>
       </header>
+
       <main>
-        {/* Wave 01: Render one ChatEntry component
-        Wave 02: Render ChatLog component */}
+        <h2> 
+        { totalLiked } ❤️s
+        </h2>
+    
+        <ChatLog
+          entries = { chatMessageList }
```

Comment:
Very minor but for React props, it's standard to omit the space between the prop and its value and also surrounding the variable name,.e.g
```javascript
entries={chatMessageList}
```